doc/releases/release-notes-3.6.rst

```diff
@@ -456,8 +479,6 @@ Libraries / Subsystems
 
 * LoRa/LoRaWAN
 
-* CAN ISO-TP
-
```

Comment:
@martinjaeger I don't think we've made any changes to ISO-TP for v3.6.0 worth mentioning here?

Reply:
That's right.